surfsense_backend/app/routes/documents_routes.py

```
@@ -191,36 +190,74 @@ async def process_file_in_background(
-            loader = UnstructuredLoader(
-                file_path,
-                mode="elements",
-                post_processors=[],
-                languages=["eng"],
-                include_orig_elements=False,
-                include_metadata=False,
-                strategy="auto",
-            )
+            if app_config.ETL_SERVICE == "UNSTRUCTURED":
+                from langchain_unstructured import UnstructuredLoader
+                
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)